docs/konnector_declaration_v3.md

```diff
@@ -0,0 +1,63 @@
+# Konnector declaration proposition in V3 (Gozy)
+
+With the cozy architecture change, the konnectors will also need some adaptation to work in the V3. The goal here is to show
+to the community a proposition of declaration of a konnector in the new stack. This proposition is still to be discussed and all 
+remarks are welcome.
+
+## Constraints
+
+- For legal reasons, we need to separate the konnectors so that the user needs to decide himself to install a konnector 
+  and not the whole package (like the applications).
+- We want to propose a curated list of konnectors, but we also want to let the user install konnectors from other 
+  sources (github, npm, etc)
+- The konnectors will be executed in a container with the last LTS nodejs version (6 at the moment) with cozy-client-js giving access to the cozy stack and with access to internet (of course!)
+
+## Proposition
+
+ - The konnectors will be installed using npm. Any source usable by npm is possible : github, gist, gitlab, bitbucket (git) 
+   or a tar.gz somewhere on the internet (but not file addresse). More information here : https://docs.npmjs.com/cli/install
+ - A konnector package will need the following items :
+    - An index.js which will be run by the stack in a securised container
```

Comment:
We can use the `main` field from `package.json` and fallback to `index.js` if missing